tests/Unit/CLI/VersionTest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Arkitect\Tests\Unit\CLI;
+
+use Arkitect\CLI\Version;
+use PHPUnit\Framework\TestCase;
+
+class VersionTest extends TestCase
+{
+    public function test_it_should_return_version(): void
+    {
+        $this->assertEquals('0.1.11', Version::get());
```

Comment:
This will fail when we bump the version up, is that intended?

Reply:
Every time we need to update this test. It's not the best solution, we can remove this test if it costs too much in the future.